src/main/java/mate/academy/dao/impl/RoleDaoImpl.java

```diff
@@ -12,15 +12,15 @@
 @Repository
 public class RoleDaoImpl extends AbstractDao<Role, Long> implements RoleDao {
     @Autowired
-    protected RoleDaoImpl(SessionFactory sessionFactory) {
+    public RoleDaoImpl(SessionFactory sessionFactory) {
```

Comment:
Of course you can create this class with mock object and reflexions, but l leave it to the discretion of the mentor)